src/main/java/org/jabref/gui/FXDialogService.java

```diff
@@ -68,7 +68,7 @@ private static FXDialog createDialog(AlertType type, String title, String conten
     }
 
     private static FXDialog createDialogWithOptOut(AlertType type, String title, String content,
-                                                   String optOutMessage, Consumer<Boolean> optOutAction) {
+            String optOutMessage, Consumer<Boolean> optOutAction) {
```

Comment:
This seems to be another inconsistency between eclipse and intellj style

Reply:
I will investigate that